gatewayStubConfigurations/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM andyrbell/mountebank
```

Comment:
Codacy found an issue: [Always tag the version of an image explicitly](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5933330)